test/serializedtypes-test.js

```diff
@@ -570,8 +572,10 @@ describe('Serialized types', function() {
       assert.strictEqual(types.Amount.parse(so).to_json(), '270544960');
     });
     it('Parse 1161981756646125568 XRP', function () {
-      var so = new SerializedObject('5020304050607080');
-      assert.strictEqual(types.Amount.parse(so).to_json(), '1161981756646125696');
+      assert.throws(function() {
+        var so = new SerializedObject('1161981756646125568');
```

Comment:
The original line was `var so = new SerializedObject('5020304050607080');`  which doesn't seem to map to the `Parse 1161981756646125568 XRP'`. 

May as well have it make sense?